samples/local_attestation/README.md

```diff
@@ -226,12 +226,24 @@ See [here](https://github.com/Microsoft/openenclave/tree/master/docs/MbedtlsSupp
 
 ## Build and run
 
-To build the sample, change into the local_attestation directory and run `make build` to build the sample and run `make run` to run it.
+Note that there are two different build systems supported, one using GNU Make and
+`pkg-config`, the other using CMake.
 
-For example:
+### CMake
+
+This uses the CMake package provided by the Open Enclave SDK.
+
+```bash
+cd local_attestation
+mkdir build && cd build
+cmake ..
+make run
+```
+
+### GNU CMake
```

Comment:
Ditto (and I don't' know why this one is in italics, lol GitHub)

Reply:
addressed